langchain/langchain_interceptor.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import annotations
+import asyncio
+from contextvars import copy_context
+
+from langsmith import get_tracing_context
```

Comment:
Can this dependency be explicitly added to the `langchain` dependency group in `pyproject.toml`? (even though it is probably transitively included)